arch/arm/mach-imx/mx6/Kconfig

```diff
@@ -61,7 +61,7 @@ config MX6SLL
 
 config MX6UL
 	bool "i.MX 6UltraLite SoC support"
-	select HAS_CAAM
+	select HAS_CAAM if !OPTEE
```

Comment:
In theory this shouldn't really cause any issue as long the jr device tree node is marked as reserved by the secure world (done by optee), so we should really investigate why here (good to merge as is, but please create a jira so we can track it later).

Reply:
Thanks, this fix shouldn't have been there. Fixed.